### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by nvpmai95**
- update basic setup info in README
- add folder to create a github page so that we can have a proper user manual

### CODE REVIEW

File: app/views/admin/shared/_menu_sidebar.html.erb [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <aside class="sidebar-left">
-  <%= link_to image_tag('logo.png'), root_path %>
+  <%= link_to image_tag('logo'), root_path %>
```
Comment (harley) [inline app/views/admin/shared/_menu_sidebar.html.erb]: let's be specific in the file extension. keep the .png or .jpg whenever applicable :)

File: README.md [hunk 2]
```diff
@@ -26,12 +47,54 @@ bundle install
...
 
-## Open Source
+## 4. Managing App Configuration
+All app variables are stored in `config/initializers/constants.rb`.
+### Logo
+Replace current logo with your logo with filename `logo` in folder `/app/assets/images`
 
-Why? We embrace the [Open Source By Default](http://code.dblock.org/2015/02/09/becoming-open-source-by-default.html) spirit.
-There are many inspiring examples, such as https://realm.io/news/orta-therox-moving-to-oss-by-default/ and https://hackernoon.com/ello-is-now-open-source-by-default-88aabb0d2a7#.ohsitvw4n.
+## 5. constaRun locally
```
Comment (harley) [inline README.md]: typo :)

File: README.md [hunk 3]
```diff
@@ -1,23 +1,44 @@
-## School Website
+# Blue Sky Academy's School Website (BSA)
+[![Code Climate](https://codeclimate.com/github/blueskyacademy/new-bluesky/badges/gpa.svg)](https://codeclimate.com/github/blueskyacademy/new-bluesky) [![Test Coverage](https://codeclimate.com/github/blueskyacademy/new-bluesky/badges/coverage.svg)](https://codeclimate.com/github/blueskyacademy/new-bluesky/coverage)
```
Comment (harley) [inline README.md]: add a newline after these badges

File: README.md [hunk 4]
```diff
@@ -26,12 +47,54 @@ bundle install
...
+rails c
+User.create(email: admin_email, password: your_password)
+```
+Then point your browser to `localhost:3000/admin`. Then enter your the admin's email and password to login as admin
+
+## 3. Setup Token
+Some tokens are essential for this project to run well. Some are optional.
+All the tokens are specified in [.env.sample](https://github.com/blueskyacademy/new-bluesky/blob/master/.env.sample)
+The tokens are used for these third parties:
+### [Facebook-album-browser](https://github.com/dejanstojanovic/Facebook-Album-Browser)
```
Comment (harley) [inline README.md]: don't make these tools/libraries headers. just regular text is fine

File: app/views/admin/shared/_menu_sidebar.html.erb (hunk 1 above)
Comment (nvpmai95) [inline app/views/admin/shared/_menu_sidebar.html.erb]: i did so b/c if sb replaces logo file with different extension, this still works

**Pull request merged**